client/client.go

```diff
@@ -46,6 +46,73 @@ func Home(db *sqlx.DB, ds *DarkmodeStatus) func(c *gin.Context) {
 	}
 }
 
+// Abritrary issue featuring all the articles
+func Issue(db *sqlx.DB, ds *DarkmodeStatus) func(c *gin.Context) {
+	type issueArticle struct {
+		Title       string
+		ArticleLink string
+		Authors     string
+		Content     template.HTML
+		LastEdited  string
+	}
+
+	return func(c *gin.Context) {
+		issueID, err := pathIntSeparator(c.Param("issue"))
+		if err != nil {
+			c.Redirect(http.StatusBadRequest, "")
+			return
+		}
+
+		darkmode := Darkmode(ds)
+
+		issue, err := database.GetIssue(db, issueID, darkmode)
+		if err != nil {
+			c.Redirect(http.StatusInternalServerError, "")
+			return
+		}
+
+		articles, err := database.GetArticles(db, issueID, darkmode)
+		if err != nil {
+			c.Redirect(http.StatusInternalServerError, "")
+			return
+		}
+
+		var issueArticles []issueArticle
+		for _, article := range articles {
+			var authors string
+			if article.AuthorText.Valid {
+				emptyAuthors := make([]database.Author, 0)
+				authors = authortext(article.AuthorText, emptyAuthors)
+			} else {
+				databaseAuthors, err := database.GetAuthors(db, article.ID)
```

Comment:
I will say I am not too happy about making this many queries to the database. Should perhaps look into making a single monolith query for all authors of all articles, but at least this is clear what it does.